### DISCUSSION TITLE
[ Error ]  develop on dbeaver 21.0.5 for windows 32bit

### DISCUSSION THREAD

**Discussion by QinFangYuan** [dbeaver/dbeaver#14120] [DBeaver Development] [answered]
recently，i want to use windows 32bit (not 64bit)  , but dbeaver 21.0.5 only support windows 64bit

so,i try to develop dbeaver on source code,just like in the picture below
I'm add win32 x86 setting in Tycho build at pom.xml
![image](https://user-images.githubusercontent.com/46182166/130634110-e8dff2d5-3b6c-4e1b-930a-bcaa1ff25ead.png)

The following exception occurs when i use **mvn package** 
![image](https://user-images.githubusercontent.com/46182166/130634156-7e5611d8-e884-4aaf-aca1-037e14a7c304.png)

I wonder how to handle these exceptions,thanks dbeaver team.

**Comment by serge-rider** [comment, marked as answer]
Eclipse platform no longer supports 32-bit architecture.

You could try to build with older Eclipse version (Oxygen?). But there is no guarantee that it will even compile with recent DBeaver codebase.

**Comment by QinFangYuan** [comment]
> Eclipse platform no longer supports 32-bit architecture.
> 
> You could try to build with older Eclipse version (Oxygen?). But there is no guarantee that it will even compile with recent DBeaver codebase.

ok, thanks, I see what you mean.

**Comment by QinFangYuan** [comment]
> Eclipse platform no longer supports 32-bit architecture.
> 
> You could try to build with older Eclipse version (Oxygen?). But there is no guarantee that it will even compile with recent DBeaver codebase.

My development platform is windows 64-bit,use **Eclipse IDE for RCP**
